tests/tests_geomstats/TestVisualizationDiscreteCurves.py

```diff
@@ -0,0 +1,90 @@
+"""Unit tests for visualization."""
```

Comment:
Rename: """Unit tests for visualization of discrete curves."""
Also change the name of file --> test_visualizaztion_discrete_curves.py